### ISSUE TITLE
Make it possible to only listen to select network interfaces

### ISSUE THREAD

**Issue by jas4711** [ejurgensen/forked-daapd#158]
I have several network interfaces in my machine, and I would like for forked-daapd to only listen on one of them.  This doesn't seem possible at the moment.  I would expect a new configuration token 'interfaces' taken a list of physical network interfaces and/or IP addresses to bind to.

**Comment by ejurgensen** [comment]
I agree this should be an option, but it probably won't be on top of my list, since I assume there are workarounds for this issue with iptables and the like.

**Comment by ejurgensen** [comment]
Closing this since it has since been added
